### ISSUE TITLE
Save image in sub folder

### ISSUE THREAD

**Issue by morcegon** [avocode/avocode#2287]
* **Version**: 3.6.8

* **Web/Desktop**: Desktop

* **Platform/Browser**: Ubuntu 18.04

* **Avocode e-mail**: renan@raessolucoes.com.br

When I trying to save an image on "subpath/imagename.jpg" instead save on this location, the app saves the image at root folder like "subpathimagename.jpg". I receive the alert: "Illegal characters used in filename".

**Comment by crashmajstr** [comment]
Hello @morcegon. thank you for reaching out. I am sorry for the inconvenience. Please, can you tell me more about your problem? If possible please attach some screenshot or screencast.
Let me know and we'll try to fix everything asap.

**Comment by morcegon** [comment]
Hello @crashmajstr. 
I'll post some screen's records to show my issue.

Here how I'm trying to save a image in a sub-folder (always worked). After the save button was clicked you can see the filename changing.
![peek 2019-01-11 11-24](https://user-images.githubusercontent.com/31614/51036271-bbfff380-1593-11e9-80ec-e16fdcccb51d.gif)

Here the message it showed
![peek 2019-01-11 11-25](https://user-images.githubusercontent.com/31614/51036272-bbfff380-1593-11e9-8b7f-46f0f2e8eb51.gif)

**Comment by crashmajstr** [comment]
thanks, @morcegon for sending this over, I'll ask our dev team why it's not working anymore.
I guess it's since 3.6.8 right?

**Comment by crashmajstr** [comment]
Hello, @morcegon I've just reached out to our dev team and really we changed it in the last update. We changed because some special symbols caused bugs in the paths. We weren't aware that users like you use it that way and actually it makes sense, but for now, we will leave it that way. To cheer you up a bit I'll forward this to our UX team to take a look at it and hopefully, they'll provide you with a better solution in the future.
Best regards,
Josef
